vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.cc

```diff
@@ -226,13 +220,7 @@ bool TRANSACTION_ST::loadFromJson(const std::string & json) {
   if (!error) {
     viewingId_ = d["viewingId"].GetString();
     surveyorId_ = d["surveyorId"].GetString();
-    contribution_fiat_amount_ = d["contribution_fiat_amount"].GetString();
-    contribution_fiat_currency_ = d["contribution_fiat_currency"].GetString();
-    contribution_altcurrency_ = d["contribution_altcurrency"].GetString();
     contribution_probi_ = d["contribution_probi"].GetString();
-    contribution_fee_ = d["contribution_fee"].GetString();
-    submissionStamp_ = d["submissionStamp"].GetString();
```

Comment:
we need `submissionStamp_` as this tells us when transaction was created. It's not used in the UI, but it's needed from debugging purposes and monthly statement later on